tests/core/test_versioning.py

```
@@ -0,0 +1,289 @@
+            status=VersionStatus.STABLE,
+            schema=basic_schema,
+            description="Initial version"
+        )
+
+        assert version == "1.0.0"
+        assert versioning.current_version == "1.0.0"
+        
+        info = versioning.get_version_info("1.0.0")
+        assert info.major == 1
+        assert info.status == VersionStatus.STABLE
+        assert info.created_at <= datetime.utcnow()
```

Ruff (0.8.2)

`datetime.datetime.utcnow()` used

(DTZ003)